.github/workflows/run_admission_test.yml

```diff
@@ -51,11 +42,6 @@ jobs:
           sudo chown -R $USER $HOME/.kube $HOME/.minikube
           sed "s/root/home\/$USER/g" $HOME/.kube/config > tmp; mv tmp $HOME/.kube/config
 
-      - name: Build noobaa image
-        run: |
-          cd ./noobaa-core
-          make noobaa NOOBAA_TAG=noobaa-core:admission-test
```

Comment:
Why not explicitly build it?
I think It is easier to debug in case we have a problem (step `Build noobaa image` is separated from `Build operator image`).

Reply:
If using the operator's default core image, building core image is not required. The image is fetched from the docker hub.